UKHomeOffice/cop-private-workflow-tasklist
# Redirect onboarded users

This PR fixes a circular redirect between offboarded users and the onboard page.

**TESTING:**
To test:
* ask Anil to offboard your user;
* navigate to https://www.dev.cop.homeoffice.gov.uk and you should be redirected to the `/onboard-user`;
* once you have onboarded, navigate to https://www.dev.cop.homeoffice.gov.uk/onboard-user and you should be redirected to the `/dashboard` page;
